src/handler.rs

```diff
@@ -23,9 +24,15 @@ use virtio_bindings::bindings::virtio_ring::VIRTIO_RING_F_EVENT_IDX;
 use vm_memory::bitmap::Bitmap;
 use vm_memory::mmap::NewBitmap;
 use vm_memory::{
-    FileOffset, GuestAddress, GuestAddressSpace, GuestMemoryMmap, GuestRegionMmap, MmapRegion,
+    Address, FileOffset, GuestAddress, GuestAddressSpace, GuestMemoryMmap, GuestRegionMmap,
+    MmapRegion,
 };
 
+use libc::c_void;
+use std::mem;
+use std::os::unix::io::RawFd;
+use virtio_bindings::bindings::virtio_net::VIRTIO_F_RING_PACKED;
```

Comment:
Reference to "virtio_bindings::bindings::virtio_net::VIRTIO_F_RING_PACKED" in generic vhost library seems strange, maybe we should enhance virito_bindings too.

Reply:
@jiangliu does this mean I should open another pull request to virtio_bindings and move "VIRTIO_F_RING_PACKED" higher up?

Reply:
Sorry, it's an issue of virtio-bindings, and there's already an issue for it:
https://github.com/rust-vmm/virtio-bindings/issues/4

so let's move on with current solution, otherwise you will get blocked.